### ISSUE TITLE
How to use model viewer to plan surf trip

### ISSUE THREAD

**Issue by MathewBiddle** [ioos/ioos_code_lab#99]
- [ ] What is language(s) for used in the example?
Python/web browser? 
- [ ] Is it focused on a particular module/software or an IOOS data service?
https://eds.ioos.us/map/#
- [ ] Can you provide a minimum example of the expected code and results in a notebook?
Maybe?

Please provide a detailed description of the suggested example below:

**Comment by MathewBiddle** [comment]
I have some stuff scattered about but I'll try to summarize my workflow here.
1. Look at significant wave height forecast for MD coast (38.2101 °N 75.0867 °W). Look for significant wave heights >1m or some significant increase to narrow down the investigation. This will get you to a 12-24 hour window typically.
   1. Resources in EDS include:
      1. Wave Watch III (WW3) `WW3 Wave Hgt @ NOAA WW3 Wave Height`
      2. GFS-Wave `WW3 Atlantic Height @ NOAA GFS-Wave Atlantic Wave Height`
      3. The Nearshore Wave Prediction System (NWPS) Chesapeake `NWPS AKQ Wave Hgt @ NOAA NWPS Chesapeake Wave Height`
2. Once you find a blip in wave heights, take a look at wave periods in that time period to see if it's just a storm or a decent offshore swell. Typically periods >5 seconds is a good start. That should get you down to a 12 hour window where it might be decent to surf.
3. Next look at the forecasted wave direction and wind direction/speed. Lots of nuances in these two pieces as wind direction could be bad (onshore) but if speeds are low it could be alright. If wind direction good (offshore) but winds are strong, could be tricky. wave direction informs which spots would catch the swell. Not many features on MD coast (barrier island), so direction not as important. This should get you to a 6 hour'ish window.
4. Finally, look at tides and water temp (wetsuit or trunks?), sunrise, sunset times.
5. Run this process every hour leading up to the event - forecast will get more accurate the closer to the event. I will start looking at real-time buoy obs ~12 hours before the event.

Here is a screenshot from EDS for the first bullet:
![image](https://user-images.githubusercontent.com/8480023/179245896-0fa610ea-a27d-439b-8de3-3c11bd7ccda5.png)

**Comment by MathewBiddle** [comment]
wish we had this available for fiona right now. 😢 

![image](https://user-images.githubusercontent.com/8480023/191989084-bcbfa61d-8961-4df8-82d7-7e325ce5a029.png)

**Comment by MathewBiddle** [comment]
So, there is a jupyter notebook at https://eds.ioos.us/thredds/catalog/catalog.html?dataset=IOOS-EDS-DS

I wonder if we could use siphon to search for the appropriate datasets referenced above. Once TDS datasets found we could easily pull in the respective obs and make some plots.

I'll start a new draft PR with that notebook here.

**Comment by MathewBiddle** [comment]
For the real-time data we should use the IOOS sensor map erddap!!

**Comment by MathewBiddle** [comment]
http://erddap.sensors.ioos.us/erddap/index.html

**Comment by MathewBiddle** [comment]
It's hurricane season again! How can we get this notebook working? Do we need reach out to the folks running eds?

**Comment by ocefpaf** [comment]
> It's hurricane season again! How can we get this notebook working? Do we need reach out to the folks running eds?

If we keep hitting those slow request responses, I guess so. If I recall correctly we could not get the model data b/c it was super slow, right?

**Comment by MathewBiddle** [comment]
correct. But, I haven't tried it in a few months. Maybe it's better?
